koku/api/status/model.py

```diff
@@ -95,7 +95,7 @@ def startup(self):
         logger.info('Python: %s', self.python_version)
         module_list = []
         for name, value in self.modules.items():
-            module_list.append(module_list.append(f'{name} - {value}'))
+            module_list.append(module_list.append('{name} - {value}'))
```

Comment:
Why are we appending a list with an append list operation? Also, removing the f string here will keep the values from getting populated in the string. I believe we've got a few issues in these few lines. Is the following what we are aiming for? 

```
for name, value in self.modules.items():
            module_list.append(f'{name} - {value}')
        if module_list:
            logger.info('Modules: %s', ', '.join(module_list))
        else:
            logger.info('Modules: None')
```

Reply:
I'm not sure what the overall code is trying to do. RTD was choking on the new-style string formatting. So, I was trying to fix that specific issue.

The duplication of appends looks like a copy/paste error to me.

Reply:
Okay. Since RTD seems to not be compatible with this Python 3.6 feature, then we should convert this to using one of the other Python string formatting methods. Just removing the `f` is a breaking change. I can open a bug to fix the method.